src/parameterizations/vertical/MOM_set_diffusivity.F90

```diff
@@ -252,18 +269,29 @@ subroutine set_diffusivity(u, v, h, u_h, v_h, tv, fluxes, optics, visc, dt, Kd_i
   ! local variables
   real :: N2_bot(SZI_(G))  ! Bottom squared buoyancy frequency [T-2 ~> s-2]
   real :: rho_bot(SZI_(G)) ! In situ near-bottom density [T-2 ~> s-2]
+  real :: h_bot(SZI_(G))   ! Bottom boundary layer thickness [H ~> m].
+  integer :: k_bot(SZI_(G))   ! Bottom boundary layer thickness top layer index [nondim].
 
   type(diffusivity_diags)  :: dd ! structure with arrays of available diags
 
   real, dimension(SZI_(G),SZJ_(G),SZK_(GV)) :: &
     T_f, S_f      ! Temperature and salinity [C ~> degC] and [S ~> ppt] with properties in massless layers
                   ! filled vertically by diffusion or the properties after full convective adjustment.
 
+  real, dimension(SZI_(G)) :: &
```

Comment:
It looks to me like `bbl_thick_1d` and `kbbl_1d` are never used.  If so, they could be eliminated.